src/main/java/heronarts/lx/midi/surface/MidiFighterTwister.java

```diff
@@ -764,6 +764,7 @@ public void onParameterChanged(LXParameter parameter) {
       }
     }
 
+    @SuppressWarnings("unused")
```

Comment:
I'd just delete this code, it's dead now

Reply:
Done.